index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
 <html>
 <head>
+<meta charset="UTF-8">
+<link rel="stylesheet" href = "newdle_style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
     <title>Newdle</title>
 </head>
 <body>
+    <!-- objects for background -->
+    <div class="backgrounds">
```

Comment:
Why use class over ID here?

Reply:
because I wanted to differentiate the different backgrounds by ID, when they belong to the same class of backgrounds?

Reply:
So class is like "Type" so where as an ID you'd expect to only exist once in the file for a single tag. multiple tags might share the same class.
e.g. the tags with the IDs first_background and second_backgorund could also both have class set to "background"

that would allow you to add CSS that affects all backgrounds but doesn't affect all divs but isn't duplicated by doing it multiple times for each ID